### PR TITLE
Add PH Holiday Friendlies 2018 article

### PR DESCRIPTION
**Pull request by SurfChu85**
Most of what's written here is still provisional. Mostly follows [this tournament](https://osu.ppy.sh/help/wiki/Tournaments/oPHST/2018)'s format.

~~Please don't merge until Taiko mappool selectors and mod usage rules are finalized.~~ Okay, good to go.

---

### CODE REVIEW

File: wiki/Tournaments/oPHF/2018/en.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
+---
+tags:
+- oPHF 2018
+- oPHF2018
+---
+# osu!Philippines Holiday Friendlies 2018
+
+![oPHF2018 Poster](hf2018poster.png)
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: oPHF2018 Logo
Place the image into a subfolder named `img` and then rename the file `logo.png`

File: wiki/Tournaments/oPHF/2018/en.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+- oPHF 2018
+- oPHF2018
+---
+# osu!Philippines Holiday Friendlies 2018
+
+![oPHF2018 Poster](hf2018poster.png)
+
+> It's now time for some Holiday cheer!
+
+The **osu!Philippines Holiday Friendlies 2018** is a semi-restricted, semi-open Tournament Series hosted by the osu!PH community for the osu!standard, osu!taiko, osu!catch, and the osu!mania game modes. This tournament aims to provide up and coming players an competitive environment where they have a chance at glory themselves.
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Add in the last sentence.
>It is the 2nd iteration of the osu!Philippines Holiday Friendlies.

File: wiki/Tournaments/oPHF/2018/en.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+
+This serves as the second edition of the tournament.
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Delete and replace with one above

File: wiki/Tournaments/oPHF/2018/en.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+
+## Proposed Tournament Schedule
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: ~~Proposed~~

File: wiki/Tournaments/oPHF/2018/en.md [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+
+| Event | Timestamp |
+|---:|---|
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: | --: | :-- |

File: wiki/Tournaments/oPHF/2018/en.md [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+| Registration Phase   | 2018-10-13/2018-10-28 |
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Please make the spacing 1 space (Pre and Post content)
Example: | Match Week 1 |

File: wiki/Tournaments/oPHF/2018/en.md [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+| Match Week 3         | 2018-11-24/2018-11-25 |
+| Match Week 4         | 2018-12-01/2018-12-02 |
+| Match Week 5         | 2018-12-08/2018-12-09 |
+| Match Week 6         | 2018-12-15/2018-12-16 |
+
+**Note**: Registrations may prematurely end when there are a sufficient number of registrants. The full schedule will be revealed before registrations close.
+
+## Prizes
+
+To be determined by the staff.
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Italic

File: wiki/Tournaments/oPHF/2018/en.md [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+
+## Organization
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Organisation

File: wiki/Tournaments/oPHF/2018/en.md [hunk 9]
```diff
@@ -0,0 +1,277 @@
...
+
+| Position | Member |
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Member(s)

File: wiki/Tournaments/oPHF/2018/en.md [hunk 10]
```diff
@@ -0,0 +1,277 @@
...
+| ------------ | -------------- |
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: | :-- | :-- |

File: wiki/Tournaments/oPHF/2018/en.md [hunk 11]
```diff
@@ -0,0 +1,277 @@
...
+| Executive Committee | ![][flag_PH] [Zaphirox](https://osu.ppy.sh/users/2478675)  (osu!standard), ![][flag_PH] [Deneb](https://osu.ppy.sh/users/2604212) (osu!taiko), ![][flag_PH] [Roido](http://osu.ppy.sh/users/6829103)  (osu!catch), ![][flag_PH] [SurfChu85](https://osu.ppy.sh/users/4469895) (osu!mania) |
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Management and Organisation

File: wiki/Tournaments/oPHF/2018/en.md [hunk 12]
```diff
@@ -0,0 +1,277 @@
...
+| Map Selectors (osu!mania 4K and 7K) | ![][flag_PH] [Lenfried-](https://osu.ppy.sh/users/5314573), ![][flag_PH] [JztCallMeRon](https://osu.ppy.sh/users/2471512), ![][flag_PH] [SurfChu85](https://osu.ppy.sh/users/4469895), ![][flag_PH] [Ainyan](https://osu.ppy.sh/users/3770641), ![][flag_SG] [Raveille](https://osu.ppy.sh/users/1388767). ![][flag_MY] [Cryolien](https://osu.ppy.sh/users/1626983), ![][flag_MY] [cheewee10](https://osu.ppy.sh/users/4477497) |
+| Match Referees | ![][flag_PH] [Zaphirox](https://osu.ppy.sh/users/2478675), ![][flag_PH] [SurfChu85](https://osu.ppy.sh/users/4469895), ![][flag_PH] [MapleSyrup-](https://osu.ppy.sh/users/1192936), ![][flag_PH] [samosita](https://osu.ppy.sh/users/6090289), ![][flag_PH] [ggnakacarlz](https://osu.ppy.sh/users/4075421), ![][flag_PH] [SilentVoice](https://osu.ppy.sh/users/9570552), ![][flag_PH] [-MomoX](http://osu.ppy.sh/users/2150415), ![][flag_PH] [Roido](http://osu.ppy.sh/users/6829103), ![][flag_SG] [Raveille](https://osu.ppy.sh/users/1388767), ![][flag_AU] [nonamegohst](https://osu.ppy.sh/users/4384859), ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263) |
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Referees

File: wiki/Tournaments/oPHF/2018/en.md [hunk 13]
```diff
@@ -0,0 +1,277 @@
...
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/801475)
+- [Livestream](https://www.twitch.tv/osuliveph)
+- [osu!PH Facebook Group](https://web.facebook.com/groups/OSU.Philippines/)
+
+------------------------
+
+## Participants
+
+Qualified participants will be added in this section.
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Italic

File: wiki/Tournaments/oPHF/2018/en.md [hunk 14]
```diff
@@ -0,0 +1,277 @@
...
+
+------------------------
+
+## Mappools
+
+Mappools will be revealed one week in advance of each stage.
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Italic

File: wiki/Tournaments/oPHF/2018/en.md [hunk 15]
```diff
@@ -0,0 +1,277 @@
...
+
+------------------------
+
+## Match Results
+
+All match results will be added in this section.
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Italic

File: wiki/Tournaments/oPHF/2018/en.md [hunk 16]
```diff
@@ -0,0 +1,277 @@
...
+
+- If the staff member is the one who's at fault, it is the Executive Committee's decision on which penalties the said staff member faces. This may include suspension or termination from service.
+
+- In addition, the staff are not accountable if the player is silenced or restricted during the course of the tournament. In case of the former, the silence duration will be considered as either the defaulting time, or disconnection time. Thus, if the silence lasts for more than 10 minutes, the said player is deemed to have lost the match by default.
+
+### Updates and Concerns
+
+- Most of the major updates regarding the tournament will be posted on this forum thread. All concerns and questions regarding the tournament may be posted on this thread, or via the official osu!Philippines Facebook Group.
+
+
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Remove this line, only need 1 break

File: wiki/Tournaments/oPHF/2018/en.md [hunk 17]
```diff
@@ -0,0 +1,277 @@
...
+- If the staff member is the one who's at fault, it is the Executive Committee's decision on which penalties the said staff member faces. This may include suspension or termination from service.
+
+- In addition, the staff are not accountable if the player is silenced or restricted during the course of the tournament. In case of the former, the silence duration will be considered as either the defaulting time, or disconnection time. Thus, if the silence lasts for more than 10 minutes, the said player is deemed to have lost the match by default.
+
+### Updates and Concerns
+
+- Most of the major updates regarding the tournament will be posted on this forum thread. All concerns and questions regarding the tournament may be posted on this thread, or via the official osu!Philippines Facebook Group.
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: on the discussion thread

**Comment by SurfChu85** [comment]
Everything should be good to go for now, unless there are minor stuff I missed.

File: wiki/Tournaments/oPHF/2018/en.md [hunk 18]
```diff
@@ -0,0 +1,278 @@
+---
+tags:
+- oPHF 2018
+- oPHF2018
+---
+# osu!Philippines Holiday Friendlies 2018
+
+![oPHF2018 Poster](img/logo.png)
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: [oPHF2018 Logo]

File: wiki/Tournaments/oPHF/2018/en.md [hunk 19]
```diff
@@ -0,0 +1,278 @@
...
+
+> It's now time for some Holiday cheer!
+
+The **osu!Philippines Holiday Friendlies 2018** is a semi-restricted, semi-open Tournament Series hosted by the osu!PH community for the osu!standard, osu!taiko, osu!catch, and the osu!mania game modes. This tournament aims to provide up and coming players a competitive environment where they have a chance at glory themselves. It is the 2nd iteration of the osu!Philippines Holiday Friendlies.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+|---:|:---|
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: | --: | :-- |

File: wiki/Tournaments/oPHF/2018/en.md [hunk 20]
```diff
@@ -0,0 +1,278 @@
...
+| Referees | ![][flag_PH] [Zaphirox](https://osu.ppy.sh/users/2478675), ![][flag_PH] [SurfChu85](https://osu.ppy.sh/users/4469895), ![][flag_PH] [MapleSyrup-](https://osu.ppy.sh/users/1192936), ![][flag_PH] [samosita](https://osu.ppy.sh/users/6090289), ![][flag_PH] [ggnakacarlz](https://osu.ppy.sh/users/4075421), ![][flag_PH] [SilentVoice](https://osu.ppy.sh/users/9570552), ![][flag_PH] [-MomoX](http://osu.ppy.sh/users/2150415), ![][flag_PH] [Roido](http://osu.ppy.sh/users/6829103), ![][flag_SG] [Raveille](https://osu.ppy.sh/users/1388767), ![][flag_AU] [nonamegohst](https://osu.ppy.sh/users/4384859), ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263) |
+| Streamers | ![][flag_PH] [aEverr](https://osu.ppy.sh/users/6027084), ![][flag_PH] [Ainyan](https://osu.ppy.sh/users/3770641), ![][flag_PH] [Deadpulse](https://osu.ppy.sh/users/8575527) |
+| Commentators | List to be finalized. |
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: `*to be announced.*`

File: wiki/Tournaments/oPHF/2018/en.md [hunk 21]
```diff
@@ -0,0 +1,278 @@
...
+
+## Prizes
+
+*To be determined by the staff.*
+
+## Organisation
+
+| Position | Member(s) |
+| :-- | :-- |
+| Management and Organisation | ![][flag_PH] [Zaphirox](https://osu.ppy.sh/users/2478675)  (osu!standard), ![][flag_PH] [Deneb](https://osu.ppy.sh/users/2604212) (osu!taiko), ![][flag_PH] [Roido](http://osu.ppy.sh/users/6829103)  (osu!catch), ![][flag_PH] [SurfChu85](https://osu.ppy.sh/users/4469895) (osu!mania) |
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: delete 1 space on Zaphirox's (osu!standard). Roido (osu!catch)

File: wiki/Tournaments/oPHF/2018/en.md [hunk 22]
```diff
@@ -0,0 +1,278 @@
...
+
+### Game Modes and Scoring Format
+
+- The tournament will host the following game modes: osu!standard, osu!taiko, osu!catch, osu!mania 4K, and osu!mania 7K. In addition, the tournament will use **Score v2** as its scoring format. This requires players to play on the following osu! Builds: Stable (Latest), Beta, and Cutting Edge.
+
+### Tournament Formats
+
+- The **osu!standard** division will adopt a Qualifier Round (explained below) in order to reduce the field into 32 players which will then be subjected to a **single elimination** bracket. Seeding for this division will be based on qualifier results.
+
+- The **osu!taiko** division will be subjected to a **single elimination** bracket. Seeding for this division will be based on Performance Points as of 2018-11-03 (18:00 UTC+8:00).
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: UTC+8

File: wiki/Tournaments/oPHF/2018/en.md [hunk 23]
```diff
@@ -0,0 +1,278 @@
...
+- The **osu!catch** division will adopt a Qualifier Round (explained below) and will be subjected to a **16-player double elimination** bracket. Seeding for this division will be based on qualifier results.
+
+- Both **osu!mania 4K** and **osu!mania 7K** divisions will adopt a Qualifier Round (explained below) in order to reduce the field to 16 players. Players who advance will be subjected to a **double elimination** bracket. Seeding for this division will be based on qualifier results.
+
+### Mods Used in the Tournament
+
+#### osu!standard
+
+- Rules for mod usage are indicated below. The mod will be applied if the beatmap chosen is included in a specific category in the mappool.
+
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Delete spacing.
Spacing only apply to separate headers. Apply to others if there's any.

File: wiki/Tournaments/oPHF/2018/en.md [hunk 24]
```diff
@@ -0,0 +1,278 @@
...
+	- **NoMod** - No mod is to be used at all for the beatmap.
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Make sure the list is straight down with its parent. (usually 2 space, not tab). Apply to all if there's any

File: wiki/Tournaments/oPHF/2018/en.md [hunk 25]
```diff
@@ -0,0 +1,278 @@
...
+	- **HR (Hard Rock)** – The Hard Rock mod will be used by both participants if the map chosen falls into the Hard Rock category of the mappool.
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Hard Rock -> HardRock
Apply to All

File: wiki/Tournaments/oPHF/2018/en.md [hunk 26]
```diff
@@ -0,0 +1,278 @@
...
+	- **HD (Hidden)** – The Hidden mod will be used by both participants if the map chosen falls into the Hidden category of the mappool.
+	- **DT (Double Time)** - The Double Time mod will be used by both participants if the map chosen falls into the Double Time category of the mappool.
```
Comment (fajar13k) [inline wiki/Tournaments/oPHF/2018/en.md]: Double Time -> DoubleTime
Apply to all

**Comment by SurfChu85** [comment]
Hopefully I haven't missed anything now.

File: wiki/Tournaments/oPHF/2018/en.md [hunk 27]
```diff
@@ -0,0 +1,220 @@
...
+- oPHF 2018
+- oPHF2018
+---
+# osu!Philippines Holiday Friendlies 2018
+
+![oPHF2018 Logo](img/logo.png)
+
+> It's now time for some Holiday cheer!
+
+The **osu!Philippines Holiday Friendlies 2018** is a semi-restricted, semi-open Tournament Series hosted by the osu!PH community for the osu!standard, osu!taiko, osu!catch, and the osu!mania game modes. This tournament aims to provide up and coming players a competitive environment where they have a chance at glory themselves. It is the 2nd iteration of the osu!Philippines Holiday Friendlies.
```
Comment (TPGPL) [inline wiki/Tournaments/oPHF/2018/en.md]: The **osu!Philippines Holiday Friendlies 2018** (***oPHF2018***) is a semi-restricted, semi-open tournament series for the osu!standard, osu!taiko, osu!catch, and the osu!mania game modes hosted by the osu!PH community. This tournament aims to provide up and coming players a competitive environment where they have a chance to glory themselves. It is the 2nd installment of the osu!Philippines Holiday Friendlies.

File: wiki/Tournaments/oPHF/2018/en.md [hunk 28]
```diff
@@ -0,0 +1,220 @@
...
+
+## Match Results
+
+*All match results will be added in this section.*
+
+------------------------
+
+## Rules
+
+### Game Modes and Scoring Format
```
Comment (TPGPL) [inline wiki/Tournaments/oPHF/2018/en.md]: Add an empty line after every subheader

e.g.
```
# Header
content
```
to
```
# Header

content
```

File: wiki/Tournaments/oPHF/2018/en.md [hunk 29]
```diff
@@ -0,0 +1,220 @@
...
+  - **FreeMod** – Both contestants must use either Hidden or Hard Rock mods if the map chosen falls into the FreeMod category of the mappool.
+	
+#### osu!taiko
+- Rules for mod usage are indicated below. The mod will be applied if the beatmap chosen is included in a specific category in the mappool.
+  - **NoMod** - No mod is to be used at all for the beatmap.
+  - **HR (HardRock)** – The HardRock mod will be used by both participants if the map chosen falls into the Hard Rock category of the mappool.
+  - **HD (Hidden)** – The Hidden mod will be used by both participants if the map chosen falls into the Hidden category of the mappool.
+  - **DT (DoubleTime)** – The DoubleTime mod will be used by both participants if the map chosen falls into the Double Time category of the mappool.
+	
+#### osu!catch
```
Comment (TPGPL) [inline wiki/Tournaments/oPHF/2018/en.md]: Add an empty line after the header

**Pull request merged**